core/src/main/scala/org/broadinstitute/dsde/rawls/billing/BillingProfileManagerDAO.scala

```diff
@@ -125,24 +119,60 @@ class BillingProfileManagerDAOImpl(samDAO: SamDAO,
       )
       .flatMap {
         case true =>
-          Future.successful(
-            Seq(
-              RawlsBillingProject(
-                RawlsBillingProjectName(azureConfig.billingProjectName),
-                CreationStatuses.Ready,
-                None,
-                None,
-                azureManagedAppCoordinates = Some(
-                  AzureManagedAppCoordinates(
-                    UUID.fromString(azureConfig.azureTenantId),
-                    UUID.fromString(azureConfig.azureSubscriptionId),
-                    azureConfig.azureResourceGroupId
+          val profileApi = apiClientProvider.getProfileApi(ctx)
+          Future.successful(profileApi.listProfiles(0, 1000).getItems.asScala.toSeq)
```

Comment:
Note that we have provide a limit to the number returned. To make the code robust we should check the number returned and continue to fetch profiles until we get them all. I didn't bother with that until we decide if we want to go with this approach (as opposed to fetching each profile individually by its `billlingProfileId`). But I don't think we should submit this with the arbitrary limit.